### ISSUE TITLE
Module is not installable on latest zef and rakudo 2019.11

### ISSUE THREAD

**Issue by Altai-man** [melezhik/sparrowdo-azure-web-cert-module#1]
```
➜  ~ zef install Sparrowdo::Azure::Web::Cert
===> Searching for: Sparrowdo::Azure::Web::Cert
===> Searching for missing dependencies: Sparrowdo
===> Searching for missing dependencies: Config::Simple, Sparrow6
===> Searching for missing dependencies: Hash::Merge, JSON::Pretty
===> Testing: JSON::Pretty:ver<0.1.0>:auth<github:FROGGS>
===> Testing [OK] for JSON::Pretty:ver<0.1.0>:auth<github:FROGGS>
===> Testing: Config::Simple:auth<Sylvain Colinet>
===> Testing [OK] for Config::Simple:auth<Sylvain Colinet>
===> Testing: Hash::Merge:ver<1.0.0>:auth<github:scriptkitties>:api<1>
[Hash::Merge] Cannot merge an undefined Hash!
[Hash::Merge]   in method merge at t/02-empty-source.t line 22
[Hash::Merge] Cannot merge an undefined Hash!
[Hash::Merge]   in method merge at t/02-empty-source.t line 22
===> Testing [OK] for Hash::Merge:ver<1.0.0>:auth<github:scriptkitties>:api<1>
===> Testing: Sparrow6:ver<0.0.11>
===> Testing [OK] for Sparrow6:ver<0.0.11>
===> Testing: Sparrowdo:ver<0.1.1>
===> Testing [OK] for Sparrowdo:ver<0.1.1>
===> Testing: Sparrowdo::Azure::Web::Cert:ver<0.0.3>
[Sparrowdo::Azure::Web::Cert] ===SORRY!===
[Sparrowdo::Azure::Web::Cert] Could not find Sparrowdo::Core::DSL::Template at line 6 in:
[Sparrowdo::Azure::Web::Cert]     file#/home/koto/.zef/store/Sparrowdo-Azure-Web-Cert-0.0.3.tar.gz/Sparrowdo-Azure-Web-Cert-0.0.3
[Sparrowdo::Azure::Web::Cert]     file#/home/koto/.zef/store/Sparrowdo-0.1.1.tar.gz/Sparrowdo-0.1.1
[Sparrowdo::Azure::Web::Cert]     file#/home/koto/.zef/store/perl6-config-simple.git/6689a300f7d8c87e3622d86aac38b512b5d2afcd
[Sparrowdo::Azure::Web::Cert]     file#/home/koto/.zef/store/p6-JSON-Pretty.git/c7c90d7358e4c806ee5460dfa25cb248e75c31a5
[Sparrowdo::Azure::Web::Cert]     file#/home/koto/.zef/store/Sparrow6-0.0.11.tar.gz/Sparrow6-0.0.11
[Sparrowdo::Azure::Web::Cert]     file#/home/koto/.zef/store/Hash-Merge-1.0.0.tar.gz/Hash-Merge-1.0.0
[Sparrowdo::Azure::Web::Cert]     inst#/home/koto/.perl6
[Sparrowdo::Azure::Web::Cert]     inst#/home/koto/.rakudobrew/versions/moar-master/install/share/perl6/site
[Sparrowdo::Azure::Web::Cert]     inst#/home/koto/.rakudobrew/versions/moar-master/install/share/perl6/vendor
[Sparrowdo::Azure::Web::Cert]     inst#/home/koto/.rakudobrew/versions/moar-master/install/share/perl6/core
[Sparrowdo::Azure::Web::Cert]     ap#
[Sparrowdo::Azure::Web::Cert]     nqp#
[Sparrowdo::Azure::Web::Cert]     perl5#
===> Testing [FAIL]: Sparrowdo::Azure::Web::Cert:ver<0.0.3>
Aborting due to test failure: Sparrowdo::Azure::Web::Cert:ver<0.0.3> (use --force-test to override)
```

**Comment by melezhik** [comment]
@Altai-man almost all `Sparrowdo::` modules now are not compatible with the latest version of Sparrowdo and so need to be updated. Let me know if it's urgent.

**Comment by melezhik** [comment]
fixed in 0.0.4 version.

https://travis-ci.org/melezhik/sparrowdo-azure-web-cert-module/builds/658385660
